src/ViewLayers/ViewLayerAnchor.vala

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2019-2021 Alecaddd (https://alecaddd.com)
+ *
+ * This file is part of Akira.
+ *
+ * Akira is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+
+ * Akira is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+
+ * You should have received a copy of the GNU General Public License
+ * along with Akira. If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Authored by: Giacomo "giacomoalbe" Alberini <giacomoalbe@gmail.com>
+ */
+
+public class Akira.ViewLayers.ViewLayerAnchor : ViewLayer {
+    public const double UI_LINE_WIDTH = 4.0;
```

Comment:
I actually think this should be part of the ViewLayerNob. Perhaps "ViewLayerNob" is the wrong name, but I think it makes more sense associated with that mechanism.

Reply:
Ok got it! I just need to add another "case", in order to draw the anchor selection border around the anchor item. Would it be better to just call the file ViewLayerSelection? It does draw everything related to selection, imho it's a better fit than just referencing the Nob alone